docs/source/index.mdx

```diff
@@ -143,228 +52,123 @@ graph RL;
   end;
 
   subgraph " ";
-    databaseB[(Product<br/>data)];
-    serviceB[Products API];
+    databaseB[(Flight<br/>data)];
+    serviceB[Flights API];
     databaseB --- serviceB;
   end;
 
+  subgraph " ";
+    databaseC[(Hotel<br/>data)];
+    serviceC[Hotels API];
+    databaseC --- serviceC;
+  end;
+
   clients(Clients);
-  serviceA & serviceB -.- clients;
+  serviceA & serviceB & serviceC -.- clients;
   class clients secondary;
 ```
 
-But with an architecture like this, a client might need to communicate with _multiple_ APIs to fetch all of the data it needs. This diminishes a powerful advantage of GraphQL over traditional REST APIs.
-
-Instead, your organization should expose a [unified supergraph](https://principledgraphql.com/integrity) that lets clients fetch _all_ of the data that they need from a single endpoint:
-
-<p style="margin-bottom: 0">✅</p>
+But if you expose multiple GraphQL APIs to clients, a client might need to communicate with _multiple_ APIs to fetch all the data it needs. With a single federated graph, you preserve a powerful advantage of GraphQL over traditional REST APIs: the ability to fetch all the data you need in a single request.
```

Comment:
```suggestion
By exposing multiple GraphQL APIs to a client, it might need to communicate with _multiple_ APIs to fetch its data. With a single federated graph, you unlock a powerful advantage of GraphQL over traditional REST APIs: the ability to fetch all the data you need in a single request.
```